picard/ui/metadatabox.py

```diff
@@ -240,7 +277,14 @@ def contextMenuEvent(self, event):
                 menu.addAction(edit_tag_action)
             removals = []
             useorigs = []
+            item = self.currentItem()
             for tag in tags:
+                if tag in LOOKUP_TAGS:
+                    if item.column() == 1 or item.column() == 2:
+                        if len(tags) == 1:
+                            lookup_action = QtGui.QAction(_(u"Lookup tag"), self.parent)
```

Comment:
Please change the text to "Lookup in &Browser" to make it consistent with the other actions with similar purpose.